.github/workflows/release.yml

```diff
@@ -32,6 +32,9 @@ jobs:
         with:
           flutter-version: '3.24.1'
 
+      - name: Setup bundletool for APK generation
+        uses: amyu/setup-bundletool@f7a6fdd8e04bb23d2fdf3c2f60c9257a6298a40a
```

Comment:
I don't want to nitpick here, but https://github.com/mukeshsolanki/bundletool-action looks a little more active.

Reply:
Some differences - mukeshsolanki downloads the latest bundletool or a specified tag, amyu always downloads 1.15.1.

mukeshsolanki might have a CI warning about using an old node version.

mukeshsolanki is configured a bit differently.

I would need to go read about how CI Actions work, but mukeshsolanki seems to be able to run bundletool for you:

https://github.com/mukeshsolanki/bundletool-action/blob/main/src/action.js#L37-L43

Versus:

https://github.com/amyu/setup-bundletool/blob/main/src/main.ts#L28-L34

Anyway, I don't feel too strongly. I went to go look because using a SHA instead of a version here wasn't typical, and saw that there are no stars or forks. mukeshsolanki is listed first on the Action store with a few stars: https://github.com/marketplace/actions/bundletool-action

So I think it's a wash! But maybe we can stick a comment about why we're using the SHA.

Reply:
I did look at that one briefly, but it requires specifying signing credentials, which made me a bit nervous, and does more than just installing bundletool so I have a feeling it might be less flexible.  This action is really simple and gives us the flexibility to use bundletool however we want, which is why I went with it.

Reply:
For actions that are at all sketchy (not very used, uncommon, etc), I like to use a SHA just to avoid them from releasing something nefarious and us automatically using it.  It just gives us more control over the code that is being run.  I looked over the source code of that commit, so I feel alright using it as-is.